letsencrypt/cli.py

```diff
@@ -64,25 +64,38 @@
 
 """
 
+plugins = plugins_disco.PluginsRegistry.find_all()
```

Comment:
Also, there should be a `TODO` comment somewhere in the existing code, that says we're missing logging for plugins... because they are initialized before logging is setup. Moving this even higher in the execution makes it even harder to fix the problem.